frontend/src/component/ui/Button.scss

```diff
@@ -2,7 +2,7 @@
 @import '../styles/color';
 @import '../styles/font';
 
-button {
+.button {
```

Comment:
Made it a class so that it doesn't interfere other forms of custom buttons. What do you suggest?

Reply:
sounds good